build/buildRuby.sh

```diff
@@ -202,6 +202,9 @@ export PATH=${RUBY_DESTDIR}/bin:$PATH
 echo "Installing Bundler into Ruby ..."
 elevate ${RUBY_DESTDIR}/bin/gem install ${BASE_DIR}/source/ext/gems/bundler-1.10.6.gem
 
+echo "Installig Mysql2 into Ruby ..."
```

Comment:
The change for the test version of ruby is in build/configure. Somehow missed that file. Will add in the next iteration. Good catch!